File: ITapply/ITapply.Models/SearchObjects/LocationSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ITapply.Models.SearchObjects
{
    public class LocationSearchObject : BaseSearchObject
    {
        public string City { get; set; } = string.Empty;

        public string Country { get; set; } = string.Empty;
    }
}

File: ITapply/ITapply.UI/itapply_mobile/lib/models/search_objects/location_search_object.dart
import 'package:itapply_desktop/models/search_objects/base_search_object.dart';
import 'package:json_annotation/json_annotation.dart';

part 'location_search_object.g.dart';

@JsonSerializable()
class LocationSearchObject extends BaseSearchObject {
  String? City;
  String? Country;

  LocationSearchObject({
    this.City,
    this.Country,
    super.Page,
    super.PageSize,
    super.IncludeTotalCount,
    super.RetrieveAll,
  });

  @override
  Map<String, dynamic> toJson() => _$LocationSearchObjectToJson(this);
}